### ISSUE TITLE
plot_matrix color bar not centered on "zero" when using a symmetric color bar

### ISSUE THREAD

**Issue by Remi-Gau** [nilearn/nilearn#4293]
Taken from the experimental surface example

https://nilearn.github.io/stable/auto_examples/08_experimental/plot_surface_image_and_maskers.html#connectivity-with-a-surface-atlas-and-surfacelabelsmasker

![image](https://github.com/nilearn/nilearn/assets/6961185/dabecd65-a2ea-4615-9345-4738953bbf7b)

In this case I would expect the "white" part of the color map to match the 0 value.

I suspect this may require using some symmetric_colormap wrangling.

**Comment by bthirion** [comment]
Yes, it sounds a bit like guessing the user's intent, which is kind of endless... The obvious thing to do would be to impose the range to be symmetric in that  case ?
